libs/api/ebpf_api.cpp

```diff
@@ -2744,7 +2744,7 @@ _ebpf_enumerate_native_sections(
         std::scoped_lock pe_parse_lock(_pe_parse_mutex);
```

Comment:
Is it intentional throwing an exception if  `error_message` is null?

Reply:
I don't understand the question. Constructing a std::string from a nullptr is undefined behavior, which triggers an AV on Windows. Hope that helps.